openmls/src/treesync/node/leaf_node.rs

```diff
@@ -136,7 +137,7 @@ impl LeafNode {
         backend: &impl OpenMlsCryptoProvider<KeyStoreProvider = KeyStore>,
         signer: &impl Signer,
     ) -> Result<Self, LeafNodeGenerationError<KeyStore::Error>> {
-        Self::generate(
+        Self::generate_update(
```

Comment:
I renamed this because `generate_update` makes it more clear that the leaf node source is, in fact, update.